xapian_db.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake"
   s.add_development_dependency "ruby-progressbar"
   s.add_development_dependency "resque",           ">= 1.19.0"
-  s.add_development_dependency "sidekiq",          ">= 2.13.0"
+  s.add_development_dependency "sidekiq",          "~> 7.1"
```

Comment:
it'll not support sidekiq 7.2.. and greater.. is it intentional?

Reply:
It will support anything `7.x` that's greater or equal `1`. But it won't support Sidekiq `8.0` - IMHO it's correct.

Reply:
you are correct. if it were 7.x.x, only then my comment was valid. all good 🎉